apps/emqx/src/emqx_schema.erl

```diff
@@ -950,13 +950,13 @@ you drop support for the insecure renegotiation, prone to MitM attacks.
        sc(boolean(),
           #{ default => Df("client_renegotiation", true)
            , desc => """
-In protocols that support client-initiated renegotiation, 
-the cost of resources of such an operation is higher for the server than the client. 
-This can act as a vector for denial of service attacks. 
-The SSL application already takes measures to counter-act such attempts, 
-but client-initiated renegotiation can be strictly disabled by setting this option to false. 
-The default value is true. Note that disabling renegotiation can result in 
-long-lived connections becoming unusable due to limits on 
+In protocols that support client-initiated renegotiation,
```

Comment:
Question: the generated markdown and html can display line breaks as a white space?

Reply:
Nopo, My intelliJ IDEA does this auto when I change the source file and save...